server/models/player.py

```diff
@@ -55,15 +56,18 @@ def _avg_salary_per_point_based_on_history(self):
 
     @property
     def _fallback_salary(self):
+        if self.fallback_salary:
+            return self.fallback_salary
+
         if self.team_id:
```

Comment:
do we still need to check the team_id here? We can always run this calculation now but I feel like we might have been using this to not give salaries to subs or something else

Reply:
That's a good point - everything seems fine without it, so I think we can simplify!